cli/crates/server/src/errors.rs

```diff
@@ -216,6 +216,10 @@ pub enum UdfBuildError {
     #[error("could not write to a temporary file '{0}': {1}")]
     CreateNotWriteToTemporaryFile(PathBuf, IoError),
 
+    /// returned if creating symlink to a file fails.
+    #[error("could not link to file: {0}")]
+    SymlinkToFile(IoError),
```

Comment:
```suggestion
    SymlinkFailure(IoError),
```
or `SymlinkCouldNotBeCreated`. The `SymlinkToFile` name doesn't say what failed, hence the need of the comment